shared_model/validators/field_validator.hpp

```diff
@@ -77,46 +87,60 @@ namespace shared_model {
           const interface::AddPeer::AddressType &address) const {
         if (not(iroha::validator::isValidIpV4(address)
```

Comment:
Are you sure that validation should look like this? Because, as I remember, `address` here is `ip:port`, not just address.
If this checks only for IPv4 or hostname compliance, then these will be failed, but should not:
```
1.1.1.1:50
domain.com:50
```

Reply:
address, in this case, will contain ip:port, or hostname. You can check out test/module/libs/validator,  there are tests for this functions